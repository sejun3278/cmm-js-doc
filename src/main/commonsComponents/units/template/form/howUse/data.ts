interface Types {
  remarks: string;
}

export const howUseContentsList: { [key: string]: Types } = {
  "사용 방법": {
    remarks: "페이지 최상위에 ",
  },
};
