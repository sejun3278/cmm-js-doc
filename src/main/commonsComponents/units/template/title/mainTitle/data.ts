// 메인 제목에 따라오는 부가설명 리스트
export const mouduleRemarksList: { [key: string]: string } = {
  Modal: `알림, 실행, 결과 등을 사용자에게 좀 더 디자인있는 UI로 표현할 수 있습니다.`,
};
