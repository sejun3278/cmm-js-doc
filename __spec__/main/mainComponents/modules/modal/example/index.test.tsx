import { render } from "@testing-library/react";
import { RecoilRoot } from "recoil";

import MyModalExample from "src/main/mainComponents/modules/modal/example/modal.example.template";
import { modalExampleInitProps } from "src/main/mainComponents/modules/modal/example/modal.example.render.data";

export default describe("Modal Example Page", () => {
  // 스냅샷
  test("Modal Example Page - Snapshot", () => {
    const { container } = render(
      <RecoilRoot>
        <MyModalExample
          info={{ idx: 1, buttonName: "클릭하세요." }}
          commonsProps={{
            isShow: [false],
            openModal: () => () => {},
            closeModal: () => () => {},
          }}
          addProps={modalExampleInitProps}
        >
          <></>
        </MyModalExample>
      </RecoilRoot>
    );
    expect(container).toMatchSnapshot();
  });
});
