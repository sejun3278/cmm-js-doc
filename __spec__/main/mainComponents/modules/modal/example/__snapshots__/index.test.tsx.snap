// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal Example Page Modal Example Page - Snapshot 1`] = `
<div>
  <button
    class="_button_"
    role="button_click_event"
  >
    클릭하세요.
  </button>
  <div
    class="cmm-modal-wrapper css-uwybve"
  >
    <div
      class="cmm-modal-items css-m029r6"
    >
      <button
        class="cmm-modal-close-button-wrapper css-1jzwdgn"
      >
        <div
          class="cmm-modal-close-button css-1ignqe8"
        />
      </button>
      <div
        class="cmm-modal-layout css-1vkx5dn"
      >
        <div
          class="cmm-modal-content css-1wdbmsj"
        />
      </div>
    </div>
  </div>
</div>
`;
